src/SonarQube.Client.RequestGenerator/ArgsParser.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System.Linq;
+
+namespace SonarQube.Client.RequestGenerator
+{
+    public class ArgsParser
```

Comment:
That's a super simple cmd line args parser that provides default values for missing arguments